docker-amundsen.yml

```diff
@@ -15,13 +15,20 @@ services:
           - 7687:7687
       volumes:
           - ./example/docker/neo4j/conf:/conf
+  elasticsearch:
+      image: elasticsearch:6.7.0
+      container_name: es_amundsen
+      ports:
+          - 9200:9200
       networks:
```

Comment:
@feng-tao this is the line I was referencing, it's hard to point out but the diff shows it present in the neo4j section, but then the elasticsearch entry is added in between so it assigns the network to elasticsearch but not in neo4j anymore.

Reply:
thanks @amweiss for reporting. Just commit the change in master :)